contnumber.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open('C://22BCA192//PYTHON//contact1.csv','w',newline='') as obj:
+    p=csv.writer(obj)
+    h=['name','contact']
+    p.writerow(h)
+    data=[['om',1234567890],
+          ['sai',1223456789],
+          ['ram',1234455777],
+         ['radha',1234455777],
+         ['gopal',345678902]]
+    p.writerows(data)

```

Comment:
is there any difference between writerow( ) and writerows( )?

Reply:
writerow():
                  This method writes a single row at a time. Field row can be written using this method. 
writerows(): 
                  This method is used to write multiple rows at a time.